python/sglang/srt/server_args.py

```diff
@@ -1641,6 +1641,7 @@ def add_cli_args(parser: argparse.ArgumentParser):
                 "flashinfer_cutlass",
                 "flashinfer_mxfp4",
                 "flashinfer_cutedsl",
+                "cutlass_w4afp8",
```

Comment:
MoE runner backend should be decoupled with quant methods. We will have `--moe-runner-backend cutlass` in the future. Check our roadmap: cutlass_w4afp8.